app/views/general/search.html.erb

```diff
@@ -202,6 +202,8 @@
     <div class="results_section">
       <% if @xapian_requests_hits == 1 && @page == 1 %>
         <h2 class="foi_results"><%= _("One FOI request found") %></h2>
+      <% elsif @xapian_requests.matches_estimated > @requests_per_page %>
+        <h2 class="foi_results"><%= _("FOI requests {{start_count}} to {{end_count}} of about {{total_count}}", :start_count => ((@page-1)*@requests_per_page+1).to_s, :end_count => [@page*@requests_per_page, @xapian_requests.matches_estimated].min.to_s, :total_count =>  @xapian_requests.matches_estimated.to_s) %></h2>
```

Comment:
Logically it would be nicer to see e.g. "1 to 4 of 4" if there's only a single page, but probably not worth the overhead. Experimental commits pushed pre-cleanup so the code wasn't only on my laptop in case it overheats some more. Reverting